### ISSUE TITLE
Flag websites with custom scrollbars that are very thin

### ISSUE THREAD

**Issue by summercms** [GoogleChrome/lighthouse#13112]
See here for more details:

Open UI: [Force bad scrollbar designs to apply accessibility patches](https://github.com/openui/open-ui/issues/404)
Google Chrome: [Issue 1251575](https://bugs.chromium.org/p/chromium/issues/detail?id=1251575)

## Feature request summary

![1_kh08HW91JyQ2FMUyHGRA9w](https://user-images.githubusercontent.com/57409060/134156771-d29e9122-43ed-4f10-9368-0ffcdb226371.png)

Website developers/designers can now customize the scrollbar in the browser, using this spec: https://drafts.csswg.org/css-scrollbars-1/

Browser support is now over 97% https://caniuse.com/?search=scrollbar

### Issue

I have come across several websites that have customized the browser scrollbar. But their websites have really bad **Scroll Jank** issues!

> "Jank" is the term used to describe the visual hiccup that occurs when the system is not able to build and provide frames in time for them to be drawn to the screen at the requested cadence (60hz, or higher). Jank is most apparent when scrolling, when what should be a smoothly animated flow has hiccups, where the movement pauses along the way for one or more frames as the app takes longer to render content than the duration of a frame on the system.

Because of the JANK issues; when I try to use the `mouse wheel` to scroll down the webpage. The `jank` is preventing the page from scrolling down. So I try to grab the scrollbar with the `mouse pointer` and move the scrollbar down.

However, I have seen quite a few websites with really thin scrollbars that I end up spending several minutes trying to grab a scrollbar that is say 1-3px wide for example.

## What is the motivation or use case for changing this?

Improve web accessibility standards.

## How is this beneficial to Lighthouse?

Google Lighthouse should flag websites with **poorly designed custom scrollbars** giving a bad user experience and causing web accessibility issues.

e.g. **making the main webpage scrollbar very thin say 1-3px width.**

![image](https://user-images.githubusercontent.com/57409060/134551656-a886d5ea-e042-41a5-a0df-057ed9522cee.png)

Google Lighthouse has a set standard for [Tap targets](https://web.dev/tap-targets/) e.g. 

> Targets that are smaller than 48 px by 48 px or closer than 8 px apart fail the audit.

I'm suggesting Google Lighthouse flag websites with very thin webpage scrollbars that fail to meet a certain minimum width threshold, e.g.

> Scrollbars with a width under 5px fail the audit.

**Comment by summercms** [comment]
https://drafts.csswg.org/css-scrollbars-1/#scrollbar-width

> Implementations should use thinner scrollbars than auto. This may mean a thin variant of scrollbar provided by the platform, or a custom scrollbar thinner than the default platform scrollbar. The scrollbar should nonetheless remain wide enough to be usable. (Implementers may wish to consult WCAG 2.1 SC 2.5.5 Target Size. [WCAG21])

https://github.com/w3c/csswg-drafts/issues/6675

**Comment by summercms** [comment]
Did a quick search in `source code` search engine and found an example.

Example Website: https://paytm.com/ (not my website)

![image](https://user-images.githubusercontent.com/57409060/134680456-550c9357-4864-4a50-b5a1-72a09a10618b.png)

**Comment by summercms** [comment]
Come across another website today, link here: https://app.santiment.net/

![image](https://user-images.githubusercontent.com/57409060/134770646-0ec95cfb-997b-4ca0-b84f-839e1dc617a1.png)

Pretty much see on a daily basis websites with this issue now (think the problem is epidemic on the web).

**Comment by summercms** [comment]
Came across a different issue with a scrollbar - I thought I'd share!

![image](https://user-images.githubusercontent.com/57409060/135094817-4373f70a-df68-45bd-9287-b3ac92b407f4.png)

The scrollbar on the website I thought didn't exist due to it being almost the same colour as the web page.

The CSS Spec should also include WCAG 2.0 level AA requirements.

**Comment by patrickhulce** [comment]
Thanks @summercms these are great examples to consider! We'll be sure to discuss this in our next sync 👍

**Comment by paulirish** [comment]
I added a link to this proposal to axe-core, where all our accessibility rules originate: [Rule proposal: BP: Pages should not make scrolling UI inaccessible or difficult to use · #2538 · dequelabs/axe-core](https://github.com/dequelabs/axe-core/issues/2538#issuecomment-933878189)

We're also excited for the developments of Greg Whitworth's issues about this in the csswg. It'd be nice to address this at the spec/browser level, too.

**Comment by Jehoel** [comment]
I described this accessibility/usability issue previously here: https://github.com/GoogleChrome/lighthouse/issues/13112
